### PR TITLE
Aeproject epoch

### PR DESCRIPTION
**Pull request by vsavovlime**
Added aeproject epoch command

### CODE REVIEW

File: bin/aeproject/commands.js [hunk 1]
```diff
@@ -25,12 +29,34 @@ const addInitOption = (program) => {
...
+      await testConfig.run(options.path);
+    })
+}
+
+const addEpochOption = (program) => {
+  program
+    .command('epoch')
+    .description('Running the epoch. Without any argument epoch will be runned with --start argument')
+    .option('-p, --stop', 'Stop the epoch')
+    .option('-s, --start', 'Start the epoch')
```
Comment (maptuhec) [inline bin/aeproject/commands.js]: not sure that we need the short versions of the commands

File: AEPROJECT.md [hunk 2]
```diff
@@ -13,6 +13,15 @@ The **init** command creates aeternity project structure with a few folders in w
...
 
+## Run unit tests
+
+```
+aeproject test
+```
+
+The **test** command help developers run their unit tests for aeternity proejcts. The command executes the tests scripts that are located in the **test** folder
+of your aeternity project.
+
```
Comment (maptuhec) [inline AEPROJECT.md]: add documentation about the epoch command

Comment (vsavovlime) [reply AEPROJECT.md]: ок

**Comment by ricricucit** [comment]
@maptuhec can I move this forward (merge)?

**Comment by maptuhec** [comment]
@ricricucit We want first to merge #118  then pull the changes from "develop" branch here, to avoid large conflicts.  I have spoken with Naz, to merge PR #118. We can ping when these PRs are ready to merge, ok?

**Pull request merged**